client/src/modules/journal/journal.js

```diff
@@ -277,6 +277,33 @@ function JournalController(Journal, Sorting, Grouping,
   // API register function
   function onRegisterApi(gridApi) {
     vm.gridApi = gridApi;
+    
+    vm.gridApi.edit.on.afterCellEdit($scope, function(rowEntity, colDef, newValue, oldValue){
+      if(newValue != oldValue) {
+        propagate(colDef.field,newValue);
+      }
+    });
+  }
+
+  function propagate(column, value){
+    var propagateColumn = ['trans_date', 'entity_uuid', 'origin_id'];
+    
+    //Check if the column updated must be propragated in all transaction
+    var checkPropagate = (propagateColumn.indexOf(column));
```

Comment:
A clearer way to do write this is to do the ` value === -1` check here.  Like this:
```js
var hasSharedProperty = propagateColumn.indexOf(column) !== -1;

if (hasSharedProperty) {
  // propagate changes to all rows!
}
```